packages/utils/src/decorators/inject-entity-manager.ts

```diff
@@ -1,5 +1,7 @@
 import { Context, SharedContext } from "@medusajs/types"
+import { MedusaContextType } from "../modules-sdk/decorators"
 
+// @deprecated Use InjectManager instead
```

Comment:
we can't deprecate this one, it is only used in the old module and they can't use the new InjectManager because there is no notion of freshManager no? maybe we should just warn to not use it at all and then remove it once the old modules are migrated? such as the inventory and stocjk

Reply:
we won't use it anymore and we must replace these modules asap. I don't see any issue flagging it as deprecated.

Reply:
there is no issue, but you can't use InjectManager in the old module, I just want to be sure that people internally start to change it for InjectManager in the old module before migrating them